src/scenes/creator/base/bread.tsx

```diff
@@ -22,30 +18,20 @@ export function Bread() {
   const breadVariants = useSelector(breadVariantsSelector);
   const bread = useSelector(breadSelector);
 
-  const breadInfo: CarouselItemInfo[] = breadVariants.map((variant, index) => ({
-    text: variant,
-    icon: icons[index],
-    index: index,
-  }));
-
-  const handleChange = (index: number, value: number) => {
+  const handleChange = (index: number, value: string) => {
```

Comment:
a common convention is that if you don't use an argument which is provided to a function you prefix it with an underscore, so in this case: 

`const handleChange = (_index: number, value: string) => {`

Reply:
fixed